File: 3_24/3task.js
function EvenDigits(number) {
    let count = 0;
    while (number > 0) {
        let digit = number % 10;
        if (digit % 2 === 0) {
            count++;
        }
        number = Math.floor(number / 10);
    }
    return count;
}

let n;
console.log("Enter the number of numbers: ");
n = parseInt(prompt());
console.log("Enter the numbers separated by a space: ");
let numbers = prompt().split(" ");
for (let i = 0; i < n; ++i) {
    let number = parseInt(numbers[i]);
    console.log(EvenDigits(number));
}
console.log();

File: 3_24/3task.cs
using System;

public class Program {
    public static int EvenDigits(int number) {
        int count = 0;
        while (number > 0) {
            int digit = number % 10;
            if (digit % 2 == 0) {
                count++;
            }
            number /= 10;
        }
        return count;
    }

    public static void Main() {
        int n;
        Console.Write("Enter the number of numbers: ");
        n = int.Parse(Console.ReadLine());
        Console.Write("Enter the numbers separated by a space: ");
        string[] numbers = Console.ReadLine().Split(' ');
        for (int i = 0; i < n; ++i) {
            int number = int.Parse(numbers[i]);
            Console.Write(EvenDigits(number));
            if (i < n - 1) {
                Console.Write(" ");
            }
        }
        Console.WriteLine();
    }
}